cli/wallet/wallet.go

```diff
@@ -180,10 +180,6 @@ func NewCommands() []cli.Command {
 						Name:  "name, n",
 						Usage: "Optional account name",
 					},
-					cli.StringFlag{
-						Name:  "contract",
-						Usage: "Verification script for custom contracts",
```

Comment:
But why? It'd not for _deployed_ contracts, it's for handcrafted non-standard verification scripts.

Reply:
Fixed.

But I still don't understand why do we need these mysterious custom verification scripts. We can't parse them, can't pre-calculate the exact fee for them, we don't know what's the parameter type for these scripts to properly fill the signing context. We can do nothing with them, just store.